src/main/java/spoon/support/comparator/DeepRepresentationComparator.java

```diff
@@ -28,7 +28,7 @@ public int compare(CtElement o1, CtElement o2) {
 		}
 		String current = getDeepRepresentation(o1);
 		String other = getDeepRepresentation(o2);
-		if (current.length() <= 0 || other.length() <= 0) {
+		if (current.length() == 0 || other.length() == 0) {
```

Comment:
?

```suggestion
		if (current.isEmpty() || other.isEmpty()) {
```

Reply:
And now @MartinWitt has to run his tool until it finds a fixpoint or cycle :rocket:

Reply:
You are correct, I had not the ability to fix style issues with a fix point algorithm.

Reply:
FYI, fixed point fixing is the main feature of https://github.com/lyxell/logifix by @lyxell 
cc/ @fermadeiral